feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt

```diff
@@ -109,6 +112,16 @@ class SessionDetailFragment : DaggerFragment() {
         binding.bottomAppBar.setOnMenuItemClickListener {
             handleNavigation(it.itemId)
         }
+
+        val windowManager = context?.getSystemService(Context.WINDOW_SERVICE) as? WindowManager
```

Comment:
Thanks! I think readable code by using variables! 👍 
By the way, Can we use inserter?

Reply:
sorry, I'm not good at English. 😭 
Do you mean code like this?
`val display = (context?.getSystemService(Context.WINDOW_SERVICE) as? WindowManager)?.defaultDisplay`

Reply:
Sorry  🙇 
Other screens use `insetter.doOnApplyWindowInsets`
https://github.com/DroidKaigi/conference-app-2020/blob/d64a1ce19837c413e5cdc0ad14d5fe202a4c0213/android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt#L99

Reply:
Thank you!! I'll fix 👀

Reply:
If this doesn't work, you may adopt your plan.

Reply:
I tried using `OnApplyWindowInsets` on `onViewCreated` and `onCreateView`, but it wasn't working. 
I'm sorry, but the problem difficult for me.
Could you decide whether to close or merge this PR? sorry... :pray: